markdown/blog/2025-Q1-marketing-report.md

```diff
@@ -0,0 +1,216 @@
+
+**Strong Interest in Version 3.0.0**
+
+The "3.0.0" documentation page held 16.6% of the total views with a solid 41s average time, showing deep interest in the newest spec release.
+
+**High Engagement Pages**
+
+"AsyncAPI document structure" and "Payload schema" had fewer visits but strong time-on-page (44s and 36s, respectively), indicating users are spending time understanding these technical sections.
+
+Similarly, "Understanding AsyncAPIs with a Practical Example" drew good engagement (28s avg.) and "Coming from OpenAPI" shows decent dwell time (30s), making them great learning pathway pages.
+
+**Notable Drops**
+
+"Generator" and "Tools" saw good traffic but experienced drops in average time spent (-13.4% and +1.9%, respectively). This might mean users don’t find these pages as intuitive or relevant and could benefit from clearer CTAs or improved content structure.
+
+## Overview of the Marketing Strategy
+
+ As highlighted in the [2024 Marketing Report](https://www.asyncapi.com/blog/2024-marketing-report), the AsyncAPI Marketing Strategy was created to guide the community's marketing efforts in a focused and impactful way. On April 1st, 2025, the updated strategy document was officially merged and is now set for implementation.
+
+Over the next few months, we will begin rolling out initiatives aligned with the strategy, and our progress will be tracked through clearly defined KPIs. and consistent quarterly reports will be shared to ensure transparency and accountability.
+
+[Read the 2025 AsyncAPI Marketing Strategy](https://github.com/asyncapi/community/blob/master/annual-goals/2025_marketing_strategy.md)
+
+## Event & Community Engagement
+
+### Google Summer of Code (GSoC) Mentorship Program
+
+- AsyncAPI has successfully joined Google Summer of Code 2025, providing mentorship opportunities to aspiring open-source contributors.
+- [A dedicated informational webinar was held on March 14](https://www.youtube.com/watch?v=R7pl1F2dYLo), led by our GSoC Program Coordinator, Elegbede (Ace) Azeez, to guide potential mentees.
+- Outcome: Increased awareness and participation in the AsyncAPI open-source ecosystem.
+
+### Community Spotlight Initiative
+
+This quarter, AsyncAPI launched the Community Spotlight Initiative to recognize outstanding contributors.
+The initiative highlights individuals making significant contributions to AsyncAPI, whether through code, documentation, advocacy, or community building.
+We published the first blog [Ashmit story](https://www.asyncapi.com/blog/community-spotlight-ashmit). This would be a monthly blog. [Find the community spotlight discussion on GitHub](https://github.com/asyncapi/community/issues/1732)
+
+### AsyncAPI Conference 2025
+
+- Speaker announcements for AsyncAPI Conference Singapore have generated positive engagement and anticipation.
+- Promotional efforts included LinkedIn and Twitter posts, and community newsletters.
+- Munich Call for Speaker is also out.
+
+
+### Impact & Next Steps
```

Comment:
Reformat this section and make the bullet points align.

Reply:
It looks aligned at my end.

Or you are talking about something else?

<img width="1440" alt="Screenshot 2025-04-11 at 15 13 23" src="https://github.com/user-attachments/assets/ef04c470-ee6d-4568-85c2-f62aedc1871a" />